lua/neogit/status.lua

```diff
@@ -617,6 +622,33 @@ local function unstage()
   refresh_status()
 end
 
+local function discard()
+  local section, item = get_current_section_item()
+
+  if section == nil or item == nil then
+    return
+  end
+
+  vim.cmd('call confirm("Do you really want to do this?", "&Yes\n&No", 2)')
```

Comment:
Why are you not using `vim.fn` here?
```suggestion
  vim.fn.confirm("Do you really want to do this?", "&Yes\n&No", 2)
```

Reply:
Brainlag. Also, while we're at it: Maybe I should actually respond to the result of the confirm dialog...